astropy/stats/sigma_clipping.py

```diff
@@ -231,12 +236,16 @@ def __init__(self, sigma=3., sigma_lower=None, sigma_upper=None,
         self.stdfunc = stdfunc
         self._cenfunc_parsed = self._parse_cenfunc(cenfunc)
         self._stdfunc_parsed = self._parse_stdfunc(stdfunc)
+        self._min_value = np.nan
+        self._max_value = np.nan
+        self._niterations = 0
         self.grow = grow
 
-        # This just checks that SciPy is available, to avoid failing later
-        # than necessary if __call__ needs it:
+        # This just checks that SciPy is available, to avoid failing
+        # later than necessary if __call__ needs it:
         if self.grow:
             from scipy.ndimage import binary_dilation
+            self.binary_dilation = binary_dilation
```

Comment:
Maybe make this a private attribute? (I thought about an try/except import on top which would set `binary_dilation = None` if scipy is not available, but think the import here, only when needed, is better, saving import time and memory for the more common case where `grow` is not used).

Reply:
done